### DISCUSSION TITLE
Ringing option for new Chat

### DISCUSSION THREAD

**Discussion by GKG4** [chatwoot/chatwoot#4699] [Feature Requests]
I am currently using LiveZilla
Like them, is it possible to add a ringing option until a new chat is accepted?
This way, if the agent is not at his/her seat, he/she hears the continuous ringing and thus picks and answers the chat.

The same can be added to the mobile app too.
It's like a call ring, once answered, can continue as a normal chat.

**Comment by iammattmartin** [comment]
I doubt this would happen as LiveZilla is a chat in time solution whereas CW is a messenger where people can come and go and still being the same message stream, including email updates.

**Comment by sojan-official** [comment]
We are tracking notification customization over here: https://github.com/chatwoot/chatwoot/issues/2983
